gulpfile.js

```diff
@@ -122,4 +122,4 @@ gulp.task('debugTranslations', function() {
   return merge(stream1, stream2);
 });
 
-gulp.task('build', ['debugTranslations', 'angular']);
+gulp.task('build', gulp.parallel(['debugTranslations', 'angular']));
```

Comment:
I looked through the Gulp 4 changelog and saw that the biggest change was the introduction of the `series` and `parallel` functions.

Are you sure that these tasks are all supposed to run in parallel? I'm having trouble trying to find the default behavior (series vs. parallel) that Gulp 3 had.

Reply:
(Asking because I couldn't find the answer regarding default behavior myself)

Reply:
I am also not 100% sure what the default behavior was. We would need to verify it with the rest of the team if anyone knows whether it should be series or parallel